source/patterns/@aws-solutions-constructs/aws-events-rule-sqs/lib/index.ts

```diff
@@ -102,50 +100,12 @@ export class EventsRuleToSqs extends Construct {
    */
   constructor(scope: Construct, id: string, props: EventsRuleToSqsProps) {
     super(scope, id);
-    defaults.CheckProps(props);
+    const convertedProps: EventsRuleToSqsProps = { ...props };
+    const wrappedConstruct: EventsRuleToSqs = new EventbridgeToSqs(this, `${id}`, convertedProps);
```

Comment:
Can't use the same ID here, there will be a name collision. We've been adding wrapped here, haven't we?

Reply:
Yes, the name was too long but it seems modifying the file name satisfies the constraints

Reply:
This change actually breaks existing usage for certain cases since suddenly the name of an Event Rule will become 7 characters longer. 

It doesn't seem like a big increase but the names are only allowed to have 64 characters (https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_Rule.html)

Reply:
Wow, good catch, thanks for pointing this out. While we can't completely alleviate this issue, we can certainly reduce the number of impacted installations by shortening what we append to 1 letter (and providing simple to implement guidance for installations still affected. Issue to address this is in I[ssue 369](https://github.com/awslabs/aws-solutions-constructs/issues/369)